### DISCUSSION TITLE
Grenade Quickswitch

### DISCUSSION THREAD

**Discussion by iBLiSSIN** [IceHacks/SurvivCheatInjector#748] [Ideas]
**Describe the feature you'd like**
Add auto quickswitch for grenades 

**Describe alternatives you've considered**
This feature is very useful 

**Additional context**
https://m.youtube.com/watch?v=MT3wCEWvSQs

**Comment by AsiaGuy** [comment]
that's a lot of damage

**Comment by IceHacks** [comment]
Switching to grenades when in range?

**Comment by AsiaGuy** [comment]
> Switching to grenades when in range?

no, that's mean you have to switch to a grenade, throw it and autotically switch back to the gun
